### PR TITLE
Added contact form

### PR DESCRIPTION
**Pull request by asifaminisonline**
1. _Added h1 tag in the Contact form
2. Added description in the Contact form
3. Added the contact form
4. Added styles to it, to make it look like a Figma template
5. Added a way to receive the Forms submitted by others_

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,7 +201,43 @@ <h3 class="about_sub_title">
...
+              <input type="email" id="email-id" placeholder="example@gmail.com" name="email-id" required/>
+            </label>
+
+            <label for="your message">
+              <textarea placeholder="Write your message here" maxlength="500" id="message" required></textarea>
+            </label>
+
+            <button type="submit" class="submit-button">
+              Get in touch
+            </button>
```
Comment (HAMMAS-SALEEM) [inline index.html]: ### Your Footer
Capture

### Figma Footer
Capture3Figma

- [ ] Your footer is missing those images which you can see in the Figma image above ⬆️ Kindly see that picture and add those missing images in your design to make it look similar to the Figma template 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @ahmadsarfrazx, @asifaminisonline :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github flow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -242,3 +242,98 @@ nav {
...
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 10px 15px;
+}
+
+.contact-bar {
+  text-align: center;
+}
+
+.btncontact {
```
Comment (ElsonOtake) [inline styles.css:315]: - [OPTIONAL] I think it's better to create a style for the button when pressed as in the Figma template. It's a project requirement to stick to the template. As not required by the previous reviewer I consider that optional.

**Review by ElsonOtake — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi , @ahmadsarfrazx @asifaminisonline 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

Minions

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**